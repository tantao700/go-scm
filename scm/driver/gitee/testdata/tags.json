[{
    "name": "v0.1",
    "commit": {
        "sha": "c5b97d5ae6c19d5c5df71a34c7fbeeda2479ccbc",
        "url": "https://gitee.com/api/v5/repos/tantao700/hello-world/commits/c5b97d5ae6c19d5c5df71a34c7fbeeda2479ccbc"
    },
    "zipball_url": "https://github.com/tantao700/hello-world/zipball/v0.1",
    "tarball_url": "https://github.com/tantao700/hello-world/tarball/v0.1"
}]