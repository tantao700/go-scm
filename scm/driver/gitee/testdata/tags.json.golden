[
    {
        "Name": "v0.0.2.3",
        "Path": "refs/tags/v1.0.0",
        "Sha": "8a8ee42ebc63516c05e60c8e531b2d9bdc008c51"
    }
]